app/src/org/commcare/activities/CommCareVerificationActivity.java

```diff
@@ -224,7 +224,10 @@ protected void onActivityResult(int requestCode, int resultCode, Intent intent)
         if (requestCode == GET_MULTIMEDIA && resultCode == Activity.RESULT_OK) {
             // we found some media, so try validating it
             newMediaToValidate = true;
+            return;
         }
+
+        super.onActivityResult(requestCode, resultCode, intent);

```

Comment:
Is super specifically _only_ supposed to get called when the result code isn't processed in this implementation?